package.json

```
@@ -19,6 +19,7 @@
     "eslint-plugin-import": "2.18.2",
     "eslint-plugin-jsx-a11y": "6.2.3",
     "eslint-plugin-no-only-tests": "2.3.1",
+    "eslint-plugin-prettier": "^3.1.2",
```

please use exact versions:

```suggestion
    "eslint-plugin-prettier": "3.1.2",
```

---

Definitely!

---

Updated